aspects/electrical/Converter/GunnsElectConverterInput.hh

```diff
@@ -99,7 +99,7 @@ class GunnsElectConverterInput : public GunnsBasicLink
         /// @brief  Minor step method for non-linear iterations.
         virtual void minorStep(const double dt, const int minorStep);
         /// @brief  Calculates flows across the link.
-        virtual void computeFlows(const double);
+        virtual void computeFlows(const double dt);
```

Comment:
By convention in the rest of the repo we keep the unused argument in the declaration, and mark it as unused in the body, see comment in the .cpp.  See minorStep as an example.  We only omit the argument name in the declaration of private & unimplemented constructors, but not in normal, implemented methods.

I don't really like our convention, and I think I would prefer just removing unused argument names like you've submitted, because it would be clearer from a casual glance at the .hh that it's unused.  However, it's important to maintain consistent coding style throughout the repo, and I don't want to make a style change to the whole repo right now.

Reply:
Reverted.
In one of the more recent versions of the C++ standard (I think 17), `[[maybe-unused]]` accomplishes the same thing and isn't compiler-specific.